File: Nível 1/1589 - Bob Conduite/main.lua
numero = tonumber(io.read())

for i = 1, numero do
    valores = io.read()
    x1, x2 = valores:match("(%S+)%s+(%S+)")
    x1 = tonumber(x1)
    x2 = tonumber(x2)
    calc = x2 + x1
    print(calc)
end

File: Nível 1/1589 - Bob Conduite/main.py
numero = int(input("Clc aq: "))

for i in range(1, numero + 1):
    valores = input("A: ")
    x1,x2 = valores.split(" ")
    x1 = int(x1)
    x2 = int(x2)
    calc = x2 + x1
    print(calc)